websites/Q/Quran.com/presence.ts

```diff
@@ -0,0 +1,86 @@
+const presence = new Presence({
+		clientId: "969064871310282813"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+function getElementByXpath(path: string) {
+	return document.evaluate(
+		path,
+		document,
+		null,
+		XPathResult.FIRST_ORDERED_NODE_TYPE,
+		null
+	).singleNodeValue;
+}
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo_1024",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname } = window.location;
+	switch (pathname) {
+		case "/": {
+			presenceData.details = "Browsing the homepage...";
+			break;
+		}
+		case "/radio": {
+			presenceData.details = "Looking through radio stations...";
+			break;
+		}
+		case "/about-us": {
+			presenceData.details = "Viewing the about us page";
+			break;
+		}
+		case "/apps": {
+			presenceData.details = "Looking at Quran apps";
+			break;
+		}
+		case "/developers": {
+			presenceData.details = "Looking at the developers page";
+			break;
+		}
+		case "/privacy": {
+			presenceData.details = "Reading privacy terms";
```

Comment:
let's not mix things up people
```suggestion
			presenceData.details = "Reading privacy policy";
```